gephi/gephi-plugins
# SemanticWebImport plugin update

## New plugin or plugin update?


## What is the purpose of this plugin?
The purpose of the SemanticWebImport plugin is to allow to build a graph in Gephi from local or remote semantic data. More details can be found at https://seinecle.github.io/gephi-tutorials/generated-html/semantic-web-importer-en.html .
  
## How to test your plugin in Gephi?

1. launch gephi with the plugin installed.
2. click on "new project".
3. select the "Semantic Web Import" tab:
   3.1 select the tab "Configurations";
   3.2 click on the button "Set Configuration" (with the drop-list to its right indicating "Humans"). It should add:
        - three resource files in the "Driver" tab;
        - one "construct" expression in the "Query" tab.
4. click on the "Run" button at the bottom of the "Semantic Web Import" window.
5. it should create a graph of 170 nodes and 256 edges.

## Checklist before submission

* [ ] Did you write unit tests to test your plugin functionalities?
  - Corese (the semantic web engine embedded) is tested, but the gui of the plugin is tested only by hand until now.
